src/robusta/core/playbooks/base_trigger.py

```diff
@@ -29,7 +34,19 @@ def should_fire(self, event: TriggerEvent, playbook_id: str):
     def build_execution_event(
         self, event: TriggerEvent, sink_findings: Dict[str, List[Finding]]
     ) -> Optional[ExecutionBaseEvent]:
-        pass
+        return build_execution_event_process_pool.submit(
+            self._build_execution_event_trampoline, event, sink_findings
+        ).result()
+
+    def _build_execution_event_trampoline(self, event: TriggerEvent, sink_findings: Dict[str, List[Finding]]):
+        create_monkey_patches()
```

Comment:
One of the results of running this here is log clutter, example:

```
[runner] 2023-12-05 08:20:46.204 INFO     HTTP Request: POST https://hfzzthymmksjyasfrdef.supabase.co/rest/v1/Issues "HTTP/1.1 201 Created"
[runner] 2023-12-05 08:20:48.894 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:20:48.894 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:20:48.894 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:20:48.895 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:20:49.081 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:20:49.082 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:20:49.082 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:20:49.082 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:20:49.118 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:20:49.118 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:20:49.119 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:20:49.119 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:20:49.156 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:20:49.157 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:20:49.157 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:20:49.157 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:20:49.203 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:20:49.204 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:20:49.205 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:20:49.205 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:20:52.035 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:21:10.535 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:21:10.536 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:21:10.536 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:21:10.540 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:21:10.616 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:21:10.617 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:21:10.617 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:21:10.617 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:21:20.936 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:21:20.937 INFO     Creating yaml monkey patch
[runner] 2023-12-05 08:21:20.937 INFO     Creating kubernetes ContainerImage monkey patch
[runner] 2023-12-05 08:21:20.938 INFO     Creating kubernetes PodFailurePolicyRUle.on_pod_conditions monkey patch
[runner] 2023-12-05 08:21:21.033 INFO     Creating hikaru monkey patches
[runner] 2023-12-05 08:21:21.033 INFO     Creating yaml monkey patch
```

Is this okay?